pkgs/by-name/mcaptcha/frontend-package.json

```diff
@@ -2,7 +2,7 @@
   "name": "vanilla",
   "main": "index.js",
   "version": "1.0.0",
-  "license": "AGPL-3.0",
+  "license": "AGPL-3.0-or-later",
```

Comment:
Unrelated? Separate commit is fine though.

Reply:
Fixes evaluation for mCaptcha, to get to a successful build with the new constraint that all checks must eval without any warnings.

Reply:
Upstream PR, just FYI, is https://github.com/mCaptcha/mCaptcha/pull/162